packages/suite/src/views/settings/general/SettingsGeneral.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import { SettingsLayout } from '@settings-components';
+import { Section } from '@suite-components/Settings';
+import { Translation } from '@suite-components';
+import { useSelector } from '@suite-hooks';
+import { isDesktop, isWeb } from '@suite-utils/env';
+
+import { Language } from './Language';
+import { Fiat } from './Fiat';
+import { Labeling } from './Labeling';
+import { LabelingDisconnect } from './LabelingDisconnect';
+import { LabelingConnect } from './LabelingConnect';
+import { Tor } from './Tor';
+import { TorAddress } from './TorAddress';
+import { TorOnionLinks } from './TorOnionLinks';
+import { Theme } from './Theme';
+import { Analytics } from './Analytics';
+import { ShowLog } from './ShowLog';
+import { ClearStorage } from './ClearStorage';
+import { VersionWithUpdate } from './VersionWithUpdate';
+import { EarlyAccess } from './EarlyAccess';
+
+const SettingsGeneral = () => {
+    const { desktopUpdate, tor } = useSelector(state => ({
+        desktopUpdate: state.desktopUpdate,
+        tor: state.suite.tor,
+    }));
+
+    return (
+        <SettingsLayout data-test="@settings/index">
+            <Section title={<Translation id="TR_LOCALIZATION" />}>
+                <Language />
+                <Fiat />
+            </Section>
+
+            <Section title={<Translation id="TR_LABELING" />}>
+                <Labeling />
+                <LabelingDisconnect />
+                <LabelingConnect />
+            </Section>
+
+            {(isDesktop() || (isWeb() && tor)) && (
```

Comment:
I am not sure if I understand it properly but `(isWeb() && tor)` means that the app is running in web and at the same time is using TOR? 

I thought that TOR was only possible in the desktop app.

Maybe there is something I am missing here.  :thought_balloon:

Reply:
I was confused as you are. 

Not sure if it works and how it works but it looks like that we detect that user is using Tor outside of Suite allow him to change onion link.
`/trezor-suite/packages/suite/src/support/suite/Tor.tsx`

Reply:
It is also used in `InstallBridge` component and `AdvancedSetup` component

Reply:
Hmm, you're right, `tor` is true when you access web Suite from .onion url.

Reply:
@marekrjpolak good point I did not thing about that situation when accessing web Suit from .onion :+1: